applications/kommander-flux/2.6.2/helmrelease/cm.yaml

```
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kommander-flux-2.6.2-config-defaults
```

```suggestion
  name: ${releaseName}-config-defaults
```
Refer to other apps and lets follow the same convention here (and then use the same value in HelmRelease config overrides).